src/test/java/teammates/test/cases/browsertests/BaseUiTestCase.java

```diff
@@ -129,6 +129,25 @@ protected static Url createLocalUrl(String testFileName) throws IOException {
         return AppPage.getNewPageInstance(browser, typeOfPage);
     }
 
+    /**
+     * Logs in a page as an instructor.
+     */
+    protected <T extends AppPage> T loginAsInstructorToPage(String instructorId, AppUrl url, Class<T> typeOfPage) {
```

Comment:
The name `loginInstructorToPage` would be more consistent with the other methods.